calico-cloud/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -0,0 +1,88 @@
+---
+description: Identify vulnerabilities in workloads to Kubernetes clusters.
+---
+
+# Automatically scan images for vulnerabilities using cluster scanner
+
+## Big picture
+
+Automatically scan Kubernetes workloads for vulnerabilities using {{prodname}} Image Assurance cluster scanner.
+
+## Value
+
+{{prodname}} Image Assurances cluster scanner helps you scan workloads running inside the cluster for vulnerabilities without much configuration. With Cluster Scanner you can:
+
+- Scan kubernetes workload images for vulnerabilities automatically as and when they are deployed.
```

Comment:
> - Scan kubernetes workload images for vulnerabilities automatically **as and** when they are deployed."

There's something grammatically incorrect here (in the **bold**). You probably don't even need that piece at all. 

You can just say: 
> - Scan Kubernetes workload images for vulnerabilities automatically when they are deployed."

Note: You should also capitalize Kubernetes.

Reply:
fixed with https://github.com/tigera/docs/pull/676/commits/16d2c8cab26cc48b8ee6446e1aa77b017e8294ac